mysql-test/suite/rpl/t/rpl_connects_tried.test

```diff
@@ -0,0 +1,60 @@
+# This test references `rpl_get_master_version_and_clock`.
+--source include/have_binlog_format_mixed.inc # The test is agnostic of binlog formats.
+
+# not-master-slave.inc
+--let $rpl_skip_start_slave= 1
+--source include/master-slave.inc
+--let $rpl_server_number= 1
+--source include/rpl_stop_server.inc # shut the primary down
+--connection slave
+
+CHANGE MASTER TO master_connect_retry=1;
+--replace_result $MASTER_MYPORT MASTER_MYPORT
+--eval CHANGE MASTER 'named' TO master_host='localhost', master_port=$MASTER_MYPORT, master_user='root', master_connect_retry=2
```

Comment:
This is theoretically inconsistent if a testing machine can take seconds to run a `SELECT … FROM information_schema.SLAVE_STATUS`.
The alternative is to rely on `debug_sync`…

Reply:
solved with https://github.com/MariaDB/server/pull/3764#discussion_r1957387809